web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -83,7 +95,7 @@ h3, h4,
 
 #site-header {
     background: none;
-    border-image-source: url(images/tile-y-border.jpg);
+    border-image-source: url(images/tile-y.jpg);
```

Comment:
Worked out how to do it without loading any additional images _at all_. Thanks for the poke ;-)